### ISSUE TITLE
Research Sound Libraries and Compile User Stories for Sound Design

### ISSUE THREAD

**Issue by jiguba** [uchicago-cs/chiventure#842]
The following wiki will document the progress of implementing sound.

**Comment by jiguba** [comment]
Created a sound wiki that compiles our research notes and user stories

**Comment by Imagineham** [comment]
Added User stories 2-11 in the wiki. These user stories represent a range of potential features relating to sound in chiventure. Some stories reflect a desire to customize sound settings and other stories reflect a desire for sound effects or music at different points throughout chiventure. This list of user stories may be updated in future sprints but, as of the end of Sprint1, provides a solid starting point for new sound features.

**Comment by vivianwu00** [comment]
Edited and updated sound implementation wiki page for the Libsoundio library and descriptions/links for supported backends.

**Comment by Imagineham** [comment]
We have decided we have a sufficient number of user stories and enough information on 3 sound libraries to begin preliminary modular design and choose a library. We may or may not add to user stories in the future but these will be added to the Sound [wiki. ](https://github.com/uchicago-cs/chiventure/wiki/Sound:-Research-Implementation-and-User-Stories). We have compiled enough information about PortAudio, libSDL, and libsoundio and will meet on 4/28 to decide which library we will use.
